### PR TITLE
Fixed SobolIndicesExperiment

### PR DESCRIPTION
**Pull request by regislebrun**
Now, it deals correctly with LHSExperiment and LowDiscrepancyExperiment. The trouble was in the management of non-iid samples: we have to use the doubling dimension trick to benefit from the orthogonality of LHS and QMC samples.

### CODE REVIEW

**Comment by jschueller** [comment]
please rebase to check if the doc builds now

File: ChangeLog [hunk 1]
```diff
@@ -25,6 +25,7 @@
  * #977 (HypothesisTest::ChiSquared is bogus)
  * #982 (LinearModelTest::LinearModelDurbinWatson with several factors)
  * #983 (Bogus Normal parameter distribution)
+ * #1000 (SaltelliSensitivityAlgorithm with LowDiscrepancyExperiment produces wrong results)
```
Comment (sofianehaddad) [inline ChangeLog]: Maybe a comment concerning new Github Issues usage

Comment (sofianehaddad) [reply ChangeLog]: otherwise it is ok for me

**Comment by jschueller** [comment]
needs a rebase

File: ChangeLog (hunk 1 above)
Comment (jschueller) [reply ChangeLog]: it's in the changelog that we use github issues now

**Comment by regislebrun** [comment]
Is there anything else to do here?

**Comment by jschueller** [comment]
Seems good, just waiting on @sofianehaddad to approve

**Comment by sofianehaddad** [comment]
I already approved it ;-)

**Pull request merged**